src/unistd.rs

```diff
@@ -200,14 +200,19 @@ impl ForkResult {
 /// be created that are identical with the exception of their pid and the
 /// return value of this function.  As an example:
 ///
-/// ```no_run
-/// use nix::unistd::{fork, ForkResult};
+/// ```
+/// use nix::{sys::wait::waitpid,unistd::{fork, ForkResult, write}};
 ///
 /// match unsafe{fork()} {
 ///    Ok(ForkResult::Parent { child, .. }) => {
 ///        println!("Continuing execution in parent process, new child has pid: {}", child);
+///        waitpid(child, None).unwrap();
+///    }
+///    Ok(ForkResult::Child) => {
+///        // Unsafe to use `println!` (or `unwrap`) here. See Safety.
+///        write(libc::STDOUT_FILENO, "I'm a new child process\n".as_bytes()).ok();
```

Comment:
Should you explicitly nul-terminate this string?

Reply:
No, I don't think so. The [underlying syscall](https://docs.rs/libc-nnsdk/0.0.1/libc/fn.write.html) takes an explicit length parameter (which the nix wrapper [provides for us](https://docs.rs/nix/0.7.0/src/nix/.cargo/registry/src/github.com-1ecc6299db9ec823/nix-0.7.0/src/unistd.rs.html#311-315))

Reply:
Yeah, that's right.  Because you're writing to stdout, there's no need to nul-terminate.  stdout never ends.